htdocs/accountancy/class/bookkeeping.class.php

```diff
@@ -252,7 +252,7 @@ public function create(User $user, $notrigger = false)
 		$sql .= " FROM " . MAIN_DB_PREFIX . $this->table_element;
 		$sql .= " WHERE doc_type = '" . $this->db->escape($this->doc_type) . "'";
 		$sql .= " AND fk_doc = " . $this->fk_doc;
-		//$sql .= " AND fk_docdet = " . $this->fk_docdet;					// This field can be 0 if record is for several lines
+		$sql .= " AND fk_docdet = " . $this->fk_docdet;					// This field can be 0 if record is for several lines
```

Comment:
I think this will generate a lot of critical bug. 
Can you describe how to reproduce the initial problem you try to fix with this ?